### PR TITLE
docs: add new theme

### LINKED ISSUE #146: Add the new docs
- [x] Add news workflows
- [x] Add new theme
- [x] Migrate current docs to new docs
- [x] Remove the old docs

### PR DESCRIPTION
**Pull request by esoubiran-aneo**
fix #146

### CODE REVIEW

**Review by ddubuc — APPROVED** [review]

**Review by dbrasseur-aneo — APPROVED** [review]

**Review by dbrasseur-aneo — APPROVED** [review]

**Comment by esoubiran-aneo** [comment]
@dbrasseur-aneo I can't merge this PR. I do not have write access.

**Pull request merged**
